examples/aggregator/gcp/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
```

Comment:
Why are we using Deployment instead of Pod?

Reply:
Deployment allows scaling up and down the application. It basically controls the number of pods in an application.